tests/common/comparator.py

```diff
@@ -49,6 +49,7 @@ def s(self, b: bytes) -> str:
 
     def _whether_return(self, cmp: bool) -> bool:
         return ((self._contains == CmpType.EQUAL and not cmp)
+                or (self._contains == CmpType.CONTAINS and not cmp)
```

Comment:
I think it's not a right fix: contains means that response results should contains the user expected results. So there maybe exist some rows in response result set but not in expected result set. we could not return directly when we find some invalid rows. We should iterate the whole response result set and compare each row in expected result set with response row.